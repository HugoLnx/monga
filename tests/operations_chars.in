void main() {
	char a,b,c,e;
	int d;
	a = 10;
	b = 20;
	c = a + b;
	d = 10;
	e = d + b;
	printf("c: %d\n", c);
	printf("a+b: %d\n", a + b);
	printf("a+b+c: %d\n", a + b + c);
	printf("b-a: %d\n", b - a);
	printf("b/a: %d\n", b / a);
	printf("a*b: %d\n", a*b);
	printf("a+b*c/a: %d\n", a+b*c/a);

	printf("e: %d\n", e);
	printf("d+b: %d\n", a + b);
	printf("d+b+c: %d\n", d + b + c);
	printf("b-d: %d\n", b - d);
	printf("b/d: %d\n", b / d);
	printf("d*b: %d\n", d*b);
	printf("d+b*c/d: %d\n", d+b*c/d);
}
