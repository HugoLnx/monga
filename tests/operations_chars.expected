c: 30
a+b: 30
a+b+c: 60
b-a: 10
b/a: 2
a*b: 200
a+b*c/a: 70
e: 30
d+b: 30
d+b+c: 60
b-d: 10
b/d: 2
d*b: 200
d+b*c/d: 70
